app/domain/DolphinManager.js

```diff
@@ -77,12 +68,38 @@ export default class DolphinManager {
         log.error("Encountered error trying to write to Dolphin comm file.", err);
       },
     });
+  }
+
+  async playFile(filePath, startDolphin = true) {
+    const uniqueId = crypto.randomBytes(3 * 4).toString('hex');
+
+    const jsonString = JSON.stringify({
+      mode: this.settings.mode || "normal",
+      replay: filePath,
+      isRealTimeMode: this.settings.isRealTimeMode || false,
+      commandId: uniqueId, // Indicates to Dolphin to play new replay
+    });
+
+    await this.writeCommFile(jsonString);
 
     if (startDolphin) {
       await this.runDolphin(true);
     }
   }
 
+  async queueFiles(filePaths) {
+    const jsonString = JSON.stringify({
+      mode: "queue",
+      replay: "",
+      isRealTimeMode: false,
+      queue: filePaths.map(filePath => ({
+        path: filePath,
```

Comment:
Might be good to add these extra fields for the users that might want it

https://github.com/project-slippi/project-slippi/blob/8176c70febb46c31c914481e275e5396eb571a19/scripts/extractCombos.js#L118-L119

Reply:
you got it